daemon/main.go

```diff
@@ -219,50 +219,77 @@ func checkMinRequirements() {
-		"Policy when to allow local stack to reach local endpoints { auto | always | policy } ")
-	flags.StringVar(&kvStore, "kvstore", "", "Key-value store type")
-	flags.Var(option.NewNamedMapOptions("kvstore-opts", &kvStoreOpts, nil), "kvstore-opt", "key-value store options")
-	flags.BoolVar(&config.KeepConfig, "keep-config", false,
-		"When restoring state, keeps containers' configuration in place")
-	flags.StringVar(&labelPrefixFile, "label-prefix-file", "", "File with valid label prefixes")
-	flags.StringSliceVar(&validLabels, "labels", []string{},
-		"List of label prefixes used to determine identity of an endpoint")
-	flags.BoolVar(&enableLogstash, "logstash", false, "Enable logstash integration")
-	flags.StringVar(&logstashAddr, "logstash-agent", "127.0.0.1:8080", "Logstash agent address")
-	flags.Uint32Var(&logstashProbeTimer, "logstash-probe-timer", 10, "Logstash probe timer (seconds)")
-	flags.StringVarP(&v6Address, "node-address", "n", "", "IPv6 address of node, must be in correct format")
-	flags.BoolVar(&config.RestoreState, "restore", false,
-		"Restores state, if possible, from previous daemon")
-	flags.BoolVar(&config.KeepTemplates, "keep-templates", false,
-		"Do not restore template files from binary")
-	flags.StringVar(&config.RunDir, "state-dir", defaults.RuntimePath, "Path to directory to store runtime state")
-	flags.StringVar(&config.LibDir, "lib-dir", defaults.LibraryPath, "Path to store runtime build environment")
-	flags.StringVar(&socketPath, "socket-path", defaults.SockPath, "Sets the socket path to listen for connections")
-	flags.StringVar(&config.LBInterface, "lb", "",
-		"Enables load balancer mode where load balancer bpf program is attached to the given interface")
-	flags.BoolVar(&config.IPv4Disabled, "disable-ipv4", false, "Disable IPv4 mode")
-	flags.StringVar(&v4Prefix, "ipv4-range", "", "IPv4 prefix")
-	flags.StringVarP(&config.Tunnel, "tunnel", "t", "vxlan", "Tunnel mode vxlan or geneve, vxlan is the default")
-	flags.StringVar(&bpfRoot, "bpf-root", "", "Path to mounted BPF filesystem")
-	flags.String("access-log", "", "Path to access log of all HTTP requests observed")
-	flags.StringSlice("agent-labels", []string{}, "Additional labels to identify this agent")
-	flags.Bool("version", false, "Print version information")
-	flags.StringSliceVar(&loggers, "log-driver", []string{}, "logging endpoints to use")
-	flags.Var(option.NewNamedMapOptions("log-opts", &logOpts, nil), "log-opt", "log driver options for cilium")
+	flags.String(
+		"access-log", "", "Path to access log of all HTTP requests observed")
+	flags.StringSlice(
+		"agent-labels", []string{}, "Additional labels to identify this agent")
+	flags.StringVar(&config.AllowLocalhost,
+		"allow-localhost", AllowLocalhostAuto, "Policy when to allow local stack to reach local endpoints { auto | always | policy } ")
+	flags.StringVar(&bpfRoot,
+		"bpf-root", "", "BPF mount path filesystem")
```

Comment:
Wouldn't this be clear if it says the word "mounted", like "... mounted filesystem"?